packages/tracing/src/browser/request.ts

```diff
@@ -40,9 +40,9 @@ export interface RequestInstrumentationOptions {
 }
 
 /** Data returned from fetch callback */
-interface FetchData {
+export interface FetchData {
```

Comment:
Why do we need to export `FetchData` and `fetchCallback`?

Reply:
For tests